src/api/spec/controllers/webui/configuration_controller_spec.rb

```diff
@@ -17,4 +17,22 @@
       end
     end
   end
+
+  describe 'PATCH #update' do
+    context 'as admin' do
+      before do
+        login(admin_user)
+        patch :update, configuration: { name: 'obs', title: 'OBS', description: 'something',
```

Comment:
Done, I use other values, not the default ones, so we can see that the update action works